### ISSUE TITLE
No response

### ISSUE THREAD

**Issue by wznmickey** [DenysMb/ChatQTP-Plasmoid#8]
![](https://github.com/DenysMb/ChatQTP-Plasmoid/assets/44784663/01a023c7-4255-4202-acf3-4467f01672ce)
When I input, it has no response.

**Comment by DenysMb** [comment]
Sorry for the delay, I was away from everything this week.

It looks like it was just a temporary problem on the API.

I use the BAI Chat API for this Plasmoid: https://theb.ai/features/chat.html

I will work on errors feedback.

**Comment by wznmickey** [comment]
It works now.

**Comment by Nanco300** [comment]
No response

**Comment by DenysMb** [comment]
The API that I was using (BAI Chat, one of the providers also used by [Bavarder Chat](https://github.com/Bavarder/Bavarder)) is not free anymore and it needs to authenticate.

I unpublished the widget from the store until I find a new API to use.

In mean time, I opened a PR for the other ChatGPT extension to integrate it a little better with the Plasma guidelines (https://github.com/dark-eye/com.darkeye.chatGPT/pull/10).

I am waiting for the plugin's author to review, if I receive no response, I will probably release the fork as other widget.
